src/content/validation.md

```diff
@@ -70,46 +70,45 @@ After selecting, we can use `isFormValid$` with the async pipe in our template
 Much like we would create validators, we can create specific rules per-field
 
 ```ts
-export const isAgeValidSelector = createSelector(
+const humanAgeValid = isBetweenNumber(14, 40);
+const elfAgeValid = isBetweenNumber(80, 800);
+const tieflingAgeValid = isBetweenNumber(35, 53);
```

Comment:
We haven't shown `isBetweenNumber` yet. We should move this slide later or we should continue using some simple greater/less than comparisons to illustrate the point, since this slide is just to show how to make a selector that returns validity.